### DISCUSSION TITLE
Disable cloud sync for .pixi folder

### DISCUSSION THREAD

**Discussion by 3f6a** [prefix-dev/pixi#1893] [Ideas]
### Problem description

The .pixi or .venv folder of a project contains the virtual environment, which is usually quite large. I think pixi could follow https://github.com/mitsuhiko/rye/issues/585 here, thus disabling cloud syncing (at least for iCloud, Dropbox) of these folders. That would be helpful in case some of us want to work with a Python project in a cloud folder.

**Comment by baszalmstra** [comment]
Moved this to a discussion in the ideas section. I think this is a good idea but Im not sure when we'll get to it.!

**Comment by ghost** [comment]
I really miss this feature. Just add dual naming — ‘.pixi’ and ‘.pixi.nosync’. My personal setup is hosted on iCloud, and it forces me to maintain an extra folder for ‘.pixi’.

**Comment by pavelzw** [comment]
Wouldn't [detached-environments](https://pixi.sh/dev/reference/pixi_configuration/#detached-environments) solve this?

**Reply by ghost** [reply]
If I install .pixi in a different directory, then VS Code might not detect it.

**Reply by pavelzw** [reply]
this is a vscode issue then, though
i would expect vscode to be able to handle this
